tests/AddFeaturePolicyHeadersTest.php

```diff
@@ -32,19 +33,7 @@ public function it_wont_set_headers_if_it_is_not_enabled_in_the_config(): void
         $this->get('test-route')->assertHeaderMissing('Permissions-Policy');
     }
 
-    /** @test */
-    public function it_throws_an_invalid_policy_class_exception_when_using_an_invalid_policy(): void
```

Comment:
this is more likely an framework issue and not one that a package should check for